packages/transactions/src/TransactionService.tsx

```diff
@@ -101,13 +114,42 @@ const Transactions: FC = () => {
   };
 
   return (
-    <RemoteDataTable<Transaction>
-      columns={columns}
-      onFetchData={fetchData}
-      onRowClick={row => {
-        navigate(`/customers/customer-invoice/${row.id}`);
-      }}
-    />
+    <>
+      <Portal container={appBarButtonsRef.current}>
```

Comment:
Getting cut off when we shrink the appbar

![image](https://user-images.githubusercontent.com/35858975/131213171-577418f5-d9de-4fbd-b628-297fdeb9e8f1.png)

Added a flex on the div and then the language button goes funky also (but maybe we are moving the language button anyway? 🤔 )

Maybe in the future something like the container takes an array of buttons and it can have some sort of mechanism to use a drop down if there's too many 🤷

Reply:
bother! didn't check that. thanks for noticing
Not sure what to do with the language button. It's not in the design, and I've just bunged it in for now

I wondered about passing in an array of buttons: in Manu's mind, we simply limit the number that of buttons we use up there, and that may work for now at least

Fixed up the size and reworked the language button in 8b7c9e8